esx_service/vmdk_ops_test.py

```diff
@@ -397,14 +397,11 @@ def create_vm(si, vm_name, datastore):
         resource_pool = hosts[0].resourcePool
         logging.info("datacenter={0} vm_folder={1} hosts={2} resource_pool={3}".format(datacenter, vm_folder,
                                                                                        hosts, resource_pool))
-
-        datastore_path = '[' + datastore + '] ' + vm_name
-
         # bare minimum VM shell, no disks. Feel free to edit
         vmx_file = vim.vm.FileInfo(logDirectory=None,
                                    snapshotDirectory=None,
                                    suspendDirectory=None,
-                                   vmPathName=datastore_path)
+                                   vmPathName='[' + datastore + '] ')
```

Comment:
Please rename create_vm param  from `datastore` to `vm_path` and it will be good to go